rmgpy/kinetics/surface.pyx

```diff
@@ -570,6 +600,54 @@ cdef class SurfaceArrhenius(Arrhenius):
         return (SurfaceArrhenius, (self.A, self.n, self.Ea, self.T0, self.Tmin, self.Tmax, self.Pmin, self.Pmax,
                                    self.coverage_dependence, self.uncertainty, self.comment))
 
+    def to_cantera_kinetics(self):
+        """
+        Converts the RMG SurfaceArrhenius object to a cantera InterfaceArrheniusRate
+        Inputs are (A,b,E)  where A is in units of m^2/kmol/s, b is dimensionless, and E is in J/kmol
+
+        There are no surface falloff reaction types in Cantera, so that isn't implemented here
+        """
+
+        import cantera as ct
+
+        rate_units_dimensionality = {'1/s': 0,
+                                     's^-1': 0,
+                                     'm^2/(mol*s)': 1,
+                                     'm^4/(mol^2*s)': 2,
+                                     'cm^2/(mol*s)': 1,
+                                     'cm^4/(mol^2*s)': 2,
+                                     'm^2/(molecule*s)': 1,
+                                     'm^4/(molecule^2*s)': 2,
+                                     'cm^2/(molecule*s)': 1,
+                                     'cm^4/(molecule^2*s)': 2,
+                                     }
+
+        if self._T0.value_si != 1:
+            A = self._A.value_si / (self._T0.value_si) ** self._n.value_si
+        else:
+            A = self._A.value_si
```

Comment:
T0 isn't an attribute of ArrheniusBM (just Arrhenius), so this should be removed.
```suggestion
        A = self._A.value_si
```